ledger/error/generator/app/src/main/scala/com/daml/error/generator/app/ErrorCodeInventoryDocsGenApp.scala

```diff
@@ -188,7 +188,11 @@ object ErrorGroupTree {
     val deprecationText = e.deprecationO.fold("")(d => s"""
          |    **Deprecation**: ${d}
          |    """.stripMargin)
-    s"""${e.code}
+    val errorCodeReferenceName = s"error_code_${e.code}"
```

Comment:
You may want to prefix these labels with the output filename as well to make them unique (looking at [this CI error](https://dev.azure.com/digitalasset/daml/_build/results?buildId=96405&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=7b720ed4-ea9c-5605-45fe-aca5cec61df4&l=3422)).

Reply:
https://github.com/digital-asset/daml/pull/11913/commits/65c4cb55e7d685097725810752b8764f21106599 should fix duplicate labels. Labels were declared in one file only but that file was also included into another file. Fixed by excluding the original file from being recognized as a source document.